easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-4.0-foss-2016b.eb

```diff
@@ -0,0 +1,38 @@
+name = 'OpenFOAM'
+version = '4.0'
```

Comment:
The 4.1 that I did uses all the modules of EB used / built for the 4.0. So I unzipped the tar.gz of 4.1, loaded the modules needed, then compiled OpenFOAM simply using ./Allwmake (without extra parameters, I only performed the sources of the etc/bashrc that initialize some stuff).
In these scenario BOTH modules of QT4 and 5 are used and the compilation goes fine without any special setting that I had to do with the .eb like set the standard of c++11 to use.

The http://openfoam.org/release/4-1/ not mentions any stuff related to QT. 

For me there is some extra stuff / optimization did by EB that generates the problem. Also for the c++11 is a bit strange that on a side we have to force it and on the other no.